conda/env/specs/binstar.py

```diff
@@ -17,9 +18,12 @@
 
     from ...env.env import Environment
 
-ENVIRONMENT_TYPE = "env"
+deprecated.module("24.7", "25.9")
 
+deprecated.constant("24.7", "25.9", "ENVIRONMENT_TYPE", "env")
```

Comment:
I'm not sure I follow why this change shows up here, did [`483f36a` (#14150)](https://github.com/conda/conda/pull/14150/commits/483f36a1609fa91dff920478d11e5884b0c09a96) leak into this PR?

Reply:
Yeah, that was an error with the pull request. Thanks for catching.